tasks/pip.yml

```
@@ -35,4 +38,4 @@
   retries: 5
   delay: 10
   register: result
-  until: result is success
+  until: result is success
```

Configure your editor to always leave a blank line at the end of the file please.

```suggestion
  until: result is success

```